istioctl/cmd/analyze.go

```diff
@@ -134,9 +134,12 @@ func Analyze() *cobra.Command {
 					return err
 				}
 				_, err = client.CoreV1().Namespaces().Get(context.TODO(), namespace, v1.GetOptions{})
-				if errors.IsNotFound(err) {
-					fmt.Fprintf(cmd.ErrOrStderr(), "namespace %q not found\n", namespace)
-					return nil
+				if err != nil {
```

Comment:
Why additional nest here for err message?

Reply:
If only checking the `NotFound` error here, the other types of errors like `Forbidden` error will be escaped, and it will continue to run the rest of the logic. The nested logic here is to prevent other errors being escaped.

Reply:
I understand, but I don't think that the error should be triggered here if the analyzer client has no privilege to fetch a namespace from the cluster.

Reply:
You right. I think it doesn't matter if there are other types of errors. Thanks!